config/configgrpc/configgrpc_test.go

```diff
@@ -49,7 +50,12 @@ func TestInvalidPemFile(t *testing.T) {
 		KeepaliveParameters: nil,
 	})
 
-	assert.Equal(t, err, &os.PathError{Op: "open", Path: "/doesnt/exist", Err: unix.ENOENT})
+	expectedError := syscall.ENOENT
+	if runtime.GOOS == "windows" {
+		expectedError = syscall.Errno(0x3)
+	}
```

Comment:
It might be better to just check that an error was thrown and ignore the specific code

Reply:
The error `0x3` does not seem to intuitive, and I feel we just force the check of the specific error number in this case.

Reply:
So I like your suggestion, just to clarify :)